### PR TITLE
Remove ugorji JSON codecs

### PR DESCRIPTION
**Pull request by liggitt**
Supercedes https://github.com/openshift/origin/pull/14632

### CODE REVIEW

**Comment by liggitt** [comment]
@deads2k @stevekuznetsov PTAL

**Comment by liggitt** [comment]
[test]

File: hack/verify-generated-json-codecs.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
+
+echo "===== Verifying Generated Ugorji JSON Codecs ====="
+
+output=$(find ${OS_ROOT}/vendor/k8s.io/kubernetes -name "types.generated.go")
+if [[ -n "${output}" ]]; then
+  echo "FAILURE: Verification of existing ugorji JSON codecs failed. These should NOT exist:"
```
Comment (stevekuznetsov) [inline hack/verify-generated-json-codecs.sh]: Nit:

```
os::log::fatal "Verification of existing ugorji JSON codecs failed. These should NOT exist:\n${output}"
```

Comment (stevekuznetsov) [inline hack/verify-generated-json-codecs.sh]: newline in string is fine

**Comment by deads2k** [comment]
lgtm [merge]

**Comment by liggitt** [comment]
[severity:blocker]

**Comment by smarterclayton** [comment]
[test]

**Comment by liggitt** [comment]
service latency test failure, green otherwise
[test]

**Comment by smarterclayton** [comment]
[test] entire queue was busted

**Pull request merged**